packages/core/src/renderer/components/kube-object-details/kube-object-detail-items/metrics/deployment-details-metrics.injectable.ts

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
In general, the guideline should be that the `core` doesn't know anything about `metrics`, therefore the fact that `metrics` exports `*MetricsInjectionToken` seems bit odd. 

It would have make more sense if the `kubeObjectDetailItemInjectionToken` would be extracted to package and then all of these implementations could also be moved. 

Might be that you have already thought it, but it cascaded to something too big?

Reply:
I think it was deemed to be a too big of an effort right now to move the implementations into a package. It was decided that the old implementations would stay in the `core` because they are too entangled and the benefit of moving is not that big. We want to make these metrics replaceable but not totally remove them from the core at this point.